requirements.json

```diff
@@ -478,6 +484,41 @@
       "containerDev": { }
     }
   },
+  "xhyve": {
+    "name": "xhyve hypervisor",
+    "description": "macOS Virtualization Platform",
+    "modulePath": "model/xhyve",
+    "installable": false,
+    "detectable": true,
+    "bundle": "no",
+    "url": "https://docs.openshift.org/latest/minishift/getting-started/installing.html",
+    "fileName": "",
+    "sha256sum": "",
+    "version": "",
+    "revision": 112440,
+    "virusTotalReport": "",
+    "vendor": "macOS",
+    "messages": {
+      "info": {
+        "xhyveinfo": {
+          "message": "Follow these steps to install and enable xhyve virtualization",
+          "link": "https://docs.openshift.org/latest/minishift/getting-started/installing.html",
```

Comment:
What is the point of this message? There is nothing relevant to xhyve in this document besides the link that is contained in the "xhyvedriverinfo" message.

Reply:
Thanks, @jrichter1  I will  remove xhyveinfo